package.json

```
@@ -4,6 +4,7 @@
   "description": "<h1>Ballers-API</h1> <br> <h3>Installation (local)</h3> <p>Clone repo to your machine with Node.js installed</p> <p>Inside the cloned directory, open your terminal and run <code>npm install && npm run start</code></p> <p>Go to http://localhost:3000/api/v1/welcome</p>",
   "main": "app.js",
   "scripts": {
+    "start": "nodemon --exec babel-node app.js",
```

Nit: `nodemon` is only perfect for development and not recommended for production (https://nodemon.io/). Can we have different npm scripts for production and development? Something like this:
```suggestion
    "start": "babel-node app.js",
    "start:dev": "nodemon --exec babel-node app.js"
```